components/brave_vpn/resources/panel/state/store.ts

```diff
@@ -32,6 +32,9 @@ const observer = {
       case PurchasedState.SESSION_EXPIRED:
         store.dispatch(Actions.purchaseExpired())
         break
+      case PurchasedState.OUT_OF_CREDENTIALS:
```

Comment:
We could get `description` here and can deliver it via action's payload.

Reply:
Fixed!